Formula/diffoscope.rb

```diff
@@ -6,6 +6,7 @@ class Diffoscope < Formula
   url "https://files.pythonhosted.org/packages/82/0b/abc82afbcc451271df4bf027a6f9442f4d7a512c48e7f94d5f3e88a4bea8/diffoscope-166.tar.gz"
   sha256 "20d0b4091ae535dc7d094bf5f366e0687e0b4337a268254b11925b8e7c9ea9c4"
   license "GPL-3.0-or-later"
+  revision 1
```

Comment:
Actually, now that we've wrapped `depends_on "gnu-tar"` in an `on_macos do` block, the revision bump should be removed since this is no-op.

Or we can keep the revision bump and drop the `gnu-tar` dependency. This latter idea seems simpler; sorry for changing my mind on this. You were right the first time 😅